clang/test/SemaCXX/GH107560.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang_cc1 -std=c++20 -fsyntax-only -verify %s
```

Comment:
Can you move the test to a pre-existing file instead of adding a new one? e.g. somewhere like clang/test/SemaTemplate/pack-deduction.cpp, since this patch is affecting something that has been around for many years.

Reply:
Done.